app/posts/claim-general-mentor-training-funding/2022-06-26-general-mentor-training-grant-guidance-and-claim-form.md

```diff
@@ -0,0 +1,120 @@
+
+One of the streams of work for the mentoring team is looking into how the grants will be distributed.
+
+We plan to distribute the funding for ITT providers through DfE’s normal business processes.
+
+Because the general mentor training grant is available to more than 20,000&nbsp;schools and we need specific information for each claim, a digital form is necessary to collect the data.
+
+To test our questions and assumptions about how the service will work, we built a guidance page and a form (in Google forms).
+
+### Guidance
+
+![GOV.UK page with summary of ITT general mentor training grant](general-mentor-training-grant-guidance.png "Apply for ITT general mentor training funding")
+
+### Claim form
+
+![Google form asking user for their email address](google-form/01-email-address.png "Claim form — email address")
+
+![Google form asking user for the URN of their school](google-form/02-school.png "Claim form — school unique reference number (URN)")
+
+![Google form asking user for number of mentors, hours of classroom cover needed, and hours of classroom cover paid for by the provider](google-form/03-mentors.png "Claim form — number of mentors and training time")
+
+![Google form asking user if each mentor trained at least one trainee](google-form/04-declaration.png "Claim form — each mentor trained at least one trainee")
+
+![Google form confirmation the form has been completed](google-form/05-confirmation.png "Claim form — confirmation")
+
+### Information required for a claim
+
+#### School details
+
+So we know who is making the claim and who to pay.
+
+The amount of funding available is adjusted for location. To pay the correct amount DfE will need to know where the school is.
+
+| **Location** | **Amount** |
+|:-----:|:-----:|
+| Inner London | £1072 |
+| Outer London | £965 |
+| Fringe | £902 |
+| Rest of England | £876 |
+
+This also allows us to check compliance using data from [Register trainee teachers](https://www.register-trainee-teachers.service.gov.uk/) (Register). Schools can only claim for mentors who mentor at least one trainee each. Assuming Register collects placement data by the end of the 2024 to 2025 academic year then we can verify that the school has not claimed for more teachers than it is eligible to.
+
+#### Number of general mentors
+
+This is a basic check on the claim. Schools can claim a maximum of 20&nbsp;hours per trainee. It would also help the DfE get an understanding of how many general mentors there are.
+
+#### Hours of classroom cover so the mentor could train
+
+The aim of the grant is to help schools pay for cover so the mentor isn’t compelled to train out of school hours.
+
+To simplify the grant, schools are claiming for hours of backfill cover. Rather than specifying the costs that may have been incurred.
+
+#### Hours of classroom cover paid for by 
+
+To prevent paying schools for something already paid for, the DfE wants to deduct any hours of backfill cover paid for by the ITT provider.
```

Comment:
```suggestion
To prevent paying schools for something already paid for, the DfE wants to deduct any hours of classroom cover paid for by the ITT provider.
```